contracts/src/v0.8/dev/KeeperRegistryBase.sol

```diff
@@ -230,12 +261,21 @@ abstract contract KeeperRegistryBase is ConfirmedOwner, ExecutionPrevention, Ree
     address from,
     uint256 id,
     bytes memory performData,
-    bool useTxGasPrice
+    bool isExecution
   ) internal view returns (PerformParams memory) {
     uint256 gasLimit = s_upkeep[id].executeGas;
     (uint256 gasWei, uint256 linkEth) = _getFeedData();
-    uint256 adjustedGasWei = _adjustGasPrice(gasWei, useTxGasPrice);
-    uint96 maxLinkPayment = _calculatePaymentAmount(gasLimit, adjustedGasWei, linkEth);
+    uint256 adjustedGasWei = _adjustGasPrice(gasWei, isExecution);
+    bytes memory data = new bytes(0);
+    if (PAYMENT_MODEL == PaymentModel.OPTIMISM) {
+      if (isExecution) {
+        data = bytes.concat(msg.data, L1_FEE_DATA_PADDING);
+      } else {
+        data = bytes.concat(performData, PERFORM_DATA_PADDING);
+        data = bytes.concat(data, L1_FEE_DATA_PADDING);
```

Comment:
I'm wondering if we can simplify this by removing `PERFORM_DATA_PADDING` as it is just 4 bytes and does not make much of a difference. wdyt?